heed/src/databases/database.rs

```diff
@@ -2646,6 +2646,74 @@ impl<KC, DC, C, CDUP> Database<KC, DC, C, CDUP> {
         }
     }
 
+    /// Removes this database entirely.
+    /// The database is removed once the txn is committed.
+    ///
+    /// # Safety
+    ///
+    /// Ensure that no other copies of the database exist before calling, as
+    /// they will become invalid.
+    /// Do not remove a database if an existing transaction has modified it.
+    /// Doing so can cause database corruption or other errors.
```

Comment:
Where did you read that? Isn't the first sentence enough? Do you mean that removing a database is not thread safe?

Reply:
See https://github.com/LMDB/lmdb/blob/b8e54b4c31378932b69f1298972de54a565185b1/libraries/liblmdb/lmdb.h#L1200-L1205

```
This call is not mutex protected. Handles should only be closed by
a single thread, and only if no other threads are going to reference
the database handle or one of its cursors any further. Do not close
a handle if an existing transaction has modified its database.
Doing so can cause misbehavior from database corruption to errors
like MDB_BAD_VALSIZE (since the DB name is gone).
```

IIUC, removing a DB should be thread-safe provided that no other existing txns have modified the DB, and no other copies of the DB hande exist.

Reply:
Ok, so let's make this function `unsafe` them, unfortunately.

Reply:
Ahaha my bad. I'm on multiple fields right now 🤦